packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -54,6 +54,10 @@ const propTypes = {
   * Whether the input displays as Invalid. Use when value does not meet validation pattern.
   */
   isInvalid: PropTypes.bool,
+  /**
+   * The label of the form control children.
+   */
+  label: PropTypes.node,
```

Comment:
Should make this `ariaLabel` to be consistent with terra-form-input and DateInput.jsx.

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1182/commits/9dce482991d05af94053ae45b62b2f573e1c4466.